### DISCUSSION TITLE
Component root failure

### DISCUSSION THREAD

**Discussion by 1800joe** [noop-inc/documentation#95] [Bugs]
If component root property is missing directory build fails and deployment hangs indefinitely

**Comment by derekwolpert** [comment]
@1800joe I cannot replicate this issue on my end. Can you let me know if you are still experiencing this issue?

(screenshot below is showing both a failed build and deployment as a result of a missing component root directory)

<img width="394" alt="Screenshot 2024-02-14 at 10 16 51 AM" src="https://github.com/noop-inc/documentation/assets/48101033/4978e098-2382-4571-a8c5-195d5fdf8c77">
